modular_skyrat/modules/modular_weapons/code/energy.dm

```diff
@@ -160,6 +237,23 @@
 	stamina = 30
 	range = 6
 
+// NTX-867 Non-lethal Projectile //
+
+/obj/item/ammo_casing/energy/electrode/laser_smg
+	projectile_type = /obj/projectile/energy/electrode/knockdown
```

Comment:
wait, are these taser projectiles?

Reply:
```
/obj/projectile/energy/electrode/knockdown
	name = "electrobolt"
	icon = 'modular_skyrat/modules/modular_weapons/icons/obj/projectiles.dmi'
	icon_state = "electro_bolt"
	knockdown = 50
	stamina = 30
```
are you positive about that? It says in the code for the projectile that it has knockdown.

Reply:
You appear to be using projectile/energy/electrode/knockdown and not projectile/energy/electrode/laser_smg.